### PR TITLE
Adding Email Support for contacts

### LINKED ISSUE #19: [Rolodex] Add Email field support for Contacts
Context
* As a user, my primary form of contact is email. I want to be able to see the email addresses present in my contacts or add emails to the missing contacts So that I can easily reach out to the contacts I search via Rolodex

Already present
* Apple and Google import already pulls this value and populate it for each `Person` in the ContactList

Work
*  Add Email as a field in the `Contact` in Rolodex
*  Contact Edit modal should show `email` as a field, showing value if present and empty if absent
* Contact edit should support update to emails for a contact
* Search should look through Email field as well

### PR DESCRIPTION
**Pull request by 200016harshjain**
Closes #19

### CODE REVIEW

**Comment by 200016harshjain** [comment]
some testing screenshots for easier validation
image
image
image
image

**Comment by prashant3863** [comment]
It looks like there is a change in how email data was stored earlier and now.

For contacts with existing email it shows up like this
Screenshot 2024-04-15 at 12 12 42 PM

When I click on edit for the contact and just save it again, the formatting fixes
Screenshot 2024-04-15 at 12 12 55 PM

File: packages/account-fs/people/search.js [hunk 1]
```diff
@@ -112,6 +112,12 @@ export const SearchCapability = {
         filteredContacts.push(person)
         continue
       }
+
+      if (person.EMAIL && person.EMAIL.some(email => email.toLowerCase().includes(queryString))) {
```
Comment (prashant3863) [inline packages/account-fs/people/search.js:116]: `person.Email.some` is breaking for accounts with existing data. For new accounts it works fine
Screenshot 2024-04-15 at 12 24 13 PM

Comment (200016harshjain) [reply packages/account-fs/people/search.js:116]: have resolved this. do have a look at it

**Pull request merged**